bin/createsrcdirandfiles.js

```diff
@@ -35,13 +35,35 @@ const createSrcDirAndFiles = (details) => {
                 // Create a script to create tables 
                 const createDbFile = `${appBaseDirectory}/src/scripts/createdb.js`;
                 const createDbFileName = fs.createWriteStream(createDbFile);
-                const createDbData = dataInFiles.createDb;
+                let createDbData;
+                if(orm && orm.toLowerCase() === 'sequelize'){
+                    createDbData = dataInFiles.createDb
+                }
+                else{
+                    createDbData = dataInFiles.noOrmcreateDb
+                }
                 openAppendFile(createDbFileName.path, createDbData);
                 // Create a script to drop tables
+
                 const dropDbFile = `${appBaseDirectory}/src/scripts/dropdb.js`;
                 const dropDbFileName = fs.createWriteStream(dropDbFile);
-                const dropDbData = dataInFiles.dropDb;
+                let dropDbData;
+                if(orm && orm.toLowerCase() === 'sequelize'){
+                    dropDbData = dataInFiles.dropDb
+                }
+                else{
+                    dropDbData = dataInFiles.noOrmDropDb
+                }
                 openAppendFile(dropDbFileName.path, dropDbData);
+
+                // Create a script to query database
+                const queriesFile = `${appBaseDirectory}/src/scripts/queries.js`;
```

Comment:
This code will create `queries.js` file even at instance where we are making use of Sequelize. 
But, I will do the fix. Thanks for the work.